src/cookies.ts

```diff
@@ -155,9 +155,17 @@ export function createStorageFromOptions(
       "@supabase/ssr: createServerClient must be initialized with cookie options that specify getAll and setAll functions (deprecated, not recommended: alternatively use get, set and remove)",
     );
   } else {
-    throw new Error(
-      "@supabase/ssr: createBrowserClient in non-browser runtimes must be initialized with cookie options that specify getAll and setAll functions (deprecated: alternatively use get, set and remove)",
-    );
+    // getting cookies when there's no window but we're in browser mode can be OK, because the developer probably is not using auth functions
+    getAll = () => {
```

Comment:
should we have an info log here or would this be too annoying?

Reply:
I have no opinion really. Somehow i want it to avoid issues, somehow i don't. maybe like a `verbose` flag?

Reply:
It would be too annoying, and especially for pre-rendering mode it's gonna fire all the time.

Reply:
ack